homeassistant/components/webrtc/config_flow.py

```diff
@@ -0,0 +1,36 @@
+"""Config flow for WebRTC."""
+from __future__ import annotations
+
+from typing import Any
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DATA_RTSP_TO_WEBRTC_URL, DOMAIN
+
+
+class WebRTCConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """WebRTC config flow."""
+
+    async def async_step_user(
+        self, user_input: dict[str, Any] | None = None
+    ) -> FlowResult:
+        """Configure the RTSPtoWebRTC server url."""
+        if self._async_current_entries():
+            return self.async_abort(reason="single_instance_allowed")
+
+        if user_input is not None:
+            await self.async_set_unique_id(DOMAIN)
```

Comment:
One other thought here is to do some kind of url validation or health checking of this backend at this point when not using discovery.